### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by harikrishnansp**
available exams reviewed

### CODE REVIEW

File: components/AccountSetting.js [hunk 1]
```diff
@@ -1,35 +1,225 @@
...
+          <DatePicker
+            style={{width: 400,marginTop:10}}
+            date={this.state.dobs}
+            mode="date"
+            format="DD-MM-YYYY"
+            minDate="01-01-1975"
+            maxDate="01-01-2015"
+            confirmBtnText="Confirm"
+            cancelBtnText="Cancel"
+            customStyles={{
```
Comment (shibinrajumathew) [inline components/AccountSetting.js]: alignment

File: components/AccountSetting.js [hunk 2]
```diff
@@ -1,35 +1,225 @@
...
+              marginLeft:15,
+              marginTop:40
 
-    }
+              }
+            }}
+            onDateChange={(date) => {this.setState({dobs: date})}}
+          />
+          <Text style={[,styles.margins,styles.line]}></Text>
+              <Text style={[,styles.margins]}></Text>
```
Comment (shibinrajumathew) [inline components/AccountSetting.js]: alignment

**Review by shibinrajumathew — COMMENTED** [review]
recheck alignment of submitted files

**Comment by harikrishnansp** [comment]
solved

**Comment by shibinrajumathew** [comment]
recheck

**Pull request merged**